src/Microsoft.PowerShell.Commands.Diagnostics/CommonUtils.cs

```diff
@@ -14,6 +14,130 @@ namespace Microsoft.PowerShell.Commands.Diagnostics.Common
 {
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Get-Counter; ConfirmImpact=; SideIndicator==>}, @{Name=Tee-Object; ConfirmImpact=None; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 606
606:         $result | Should -BeNullOrEmpty
```